src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -835,14 +682,15 @@ public boolean createBulkEntityData(
 
     @Override
     public boolean createBulkLinkedData(
+            UUID entityTypeId,
             UUID linkingEntitySetId,
             Map<UUID, Map<UUID, Map<UUID, Set<Object>>>> entitiesByLinkingId ) { // linking_id/entity_set_id/property_id
         if ( !verifyElasticsearchConnection() ) { return false; }
 
         if ( !entitiesByLinkingId.isEmpty() ) {
             try {
-                String indexName = getIndexName( linkingEntitySetId );
-                String indexType = getTypeName( linkingEntitySetId );
+                String indexName = getIndexName( entityTypeId );
+                String indexType = getTypeName( entityTypeId );
 
                 BulkRequestBuilder requestBuilder = client.prepareBulk();
                 for ( Map.Entry<UUID, Map<UUID, Map<UUID, Set<Object>>>> entitiesByLinkingIdEntry : entitiesByLinkingId
```

Comment:
In getFieldMappings(  Map<UUID, Map<UUID, Set<Object>>> entityValuesByEntitySet ) for storing linked data, shouldn't we also add entity set ids?

Reply:
nope, since we now have one document per linked entity